packages/tpetra/core/src/Tpetra_Details_StaticView.cpp

```diff
@@ -265,6 +292,60 @@ resize (Kokkos::Experimental::HIPHostPinnedSpace /* space */,
 
 #endif // KOKKOS_ENABLE_HIP
 
+#ifdef KOKKOS_ENABLE_SYCL
```

Comment:
Tpetra has different implementations for different architectures.  Should these implementation actually be in Kokkos, rather than in Tpetra?

Reply:
Also, this is the same as for `Cuda` and `HIP`.